.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+
+name: Test & Build
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+
+jobs:
+  install-go-modules:
+    strategy:
+      matrix:
+        go: ['1.16.x', '1.15.x']
+    
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Go ${{ matrix.go }}
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: Test install Go modules for v${{ matrix.go }} 
+      run: go get -v . && chamber version
+
+  test:
+    strategy:
+      matrix:
+        go: ['1.16.x']
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Go 
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+    
+    - name: Test
+      run: make test
```

Comment:
Do you know what the output of make test is? Do we know what the test coverage is and whether we're satisfied with it?

Reply:
I just looked at it in the job, but don't know the coverage, should we add a coverage report (backlog ticket)?

Reply:
That looks like it's ported over 1:1 from the circleci yaml; probably something we can reasonably say "this is fine" to and sleep well tonight.

Reply:
We've tried to port 1:1 without refactoring anything. We can improve chamber release separately.